content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -88,10 +90,14 @@ $excerpt-line-height: 20;
     display: flex;
     flex-direction: column;
     padding: 12px 16px;
+    flex-grow: 1;
 
     .info-wrap {
       flex-grow: 1;
-      margin: 0 0 12px;
+    }
+
+    .context {
+      margin: 12px 0 0;
```

Comment:
Why does this margin need to move to the 2nd sibling's top margin instead of he original 1st sibling's bottom margin?